accepted/2020/better-obsoletion/obsoletions-in-net5.md

```diff
@@ -0,0 +1,213 @@
+# Obsoletions in .NET 5
+
+**PM** [Immo Landwerth](https://github.com/terrajobst)
+
+For .NET 5, [we're making](better-obsoletion.md) it more viable to obsolete APIs.
+In this document, we're describing which APIs we intend to obsolete in .NET 5 and
+which diagnostic IDs we will use for those.
+
+With `dotnet/runtime` PR [#33248](https://github.com/dotnet/runtime/pull/33248),
+the `ObsoleteAttribute` gained new `DiagnosticId` and `UrlFormat` properties. With
+these properties, obsoletions can be grouped together when multiple APIs are
+marked as obsolete for the same reason. This capability is reflected in the list
+of obsoletions below. All of these obsoletions will use the `UrlFormat` of
+`https://aka.ms/dotnet-warnings/{0}`.
+
+1. The Constrained Execution Region (CER) feature is no longer implemented
+    * DiagnosticId: `MSLIB0004`
```

Comment:
Per offline conversation I would much prefer not using any MS/Microsoft prefix for efforts like this relating to the base libraries, which are a dotnet foundation project.

Reply:
We're getting consensus around `SYSLIB`. I'll move forward with that.